catalog/tests/test_forms.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+
+from django.test import TestCase
+from django.utils import timezone
+
+from catalog.forms import RenewBookForm
+
+class RenewBookFormTest(TestCase):
+
+    
```

Comment:
Why adding this gap? Is this something in tutorial?

Reply:
I was about to add a comment here but forgot about it. I have added comment here now.